keras/backend/tensorflow_backend.py

```diff
@@ -186,6 +186,7 @@ def get_session():
             else:
                 num_thread = int(os.environ.get('OMP_NUM_THREADS'))
                 config = tf.ConfigProto(intra_op_parallelism_threads=num_thread,
+                                        inter_op_parallelism_threads=num_thread,
```

Comment:
What is this meant to achieve? Is this always the right behavior?

Reply:
This is meant to achieve reproducible keras models, with the tensorflow backend, if the environment variable `OMP_NUM_THREADS` is set to 1.

In other words, if the user sets `OMP_NUM_THREADS=1` the tensorflow backend will use no multi threading and will make models that are reproducible.  (Using multiple threads with the tensorflow backend can create issues with reproducibility, even if the random seed is set).

If you'd prefer, I can add a new environment variable, e.g. `TF_INTER_OP_PARALLELISM_THREADS`.

My goal is to be able to set both `intra_op_parallelism_threads` and `inter_op_parallelism_threads` from an environment variable, so I can set them without having to manually manage tensorflow sessions.

Does that make sense?

Reply:
@fchollet What do you think?  I could also make a new environment variable, for setting `TF_INTER_OP_PARALLELISM_THREADS`.